onnxscript/tools/transformers_models/export_phi_test.py

```diff
@@ -0,0 +1,101 @@
+import copy
+import sys
+import unittest
+
+import numpy as np
+import onnx.inliner
+import onnxruntime
+import torch
+
+import onnxscript.optimizer
+import onnxscript.rewriter
+import onnxscript.tools.training_helper
+import onnxscript.tools.transformers_models
+import onnxscript.tools.transformers_models.phi
+
+HAS_TRANSFORMERS = onnxscript.tools.transformers_models.has_transformers()
+
+
+def export_to_onnx(model, *input_tensors, optimize=True):
+    prog = torch.onnx.dynamo_export(model, *input_tensors)
+    model_proto = prog.model_proto
+    if optimize:
+        model_proto = onnxscript.optimizer.optimize(
+            model_proto,
+            num_iterations=2,
+            onnx_shape_inference=True,
+        )
+        model_proto = onnxscript.rewriter.rewrite(model_proto)
+        model_proto = onnx.inliner.inline_local_functions(model_proto)
+    return model_proto
+
+
+class TestExportPhi(unittest.TestCase):
+
+    @unittest.skipIf(sys.platform == "win32", reason="not supported yet on Windows")
+    @unittest.skipIf(not HAS_TRANSFORMERS, reason="transformers is missing")
+    def test_phi_export_cpu(self):
+        model, input_tensors, _ = onnxscript.tools.transformers_models.phi.get_phi_model()
+        input_tensors = input_tensors[0]
+        expected = model(*input_tensors)
+        proto = export_to_onnx(model, *input_tensors)
```

Comment:
Looks like this is an integration test, judging from the file name (export_phi_test not phi_test) and it tests more things than the public functions from the phi module. So I would move this file to `tests/`

Reply:
It is checking that there is no discrepancies. Is it considered as integration test?